metaflow/current.py

```diff
@@ -83,49 +87,53 @@ def is_running_flow(self) -> bool:
         return self._is_running
 
     @property
-    def flow_name(self) -> str:
+    def flow_name(self) -> Optional[str]:
         """
-        The name of the currently executing flow.
+        The name of the currently executing flow. Will be None if not inside
+        a currently executing flow.
```

Comment:
The current object is only used inside an executing flow from a user POV. This comment would indicate otherwise.

Reply:
I mean it is accessible outside and the value is sometimes None so wanted to capture that. We can remove if you want though.